tsnet/tsnet_test.go

```diff
@@ -344,3 +345,24 @@ func TestTailscaleIPs(t *testing.T) {
 			sIp4, upIp4, sIp6, upIp6)
 	}
 }
+
+func TestListenerCleanup(t *testing.T) {
```

Comment:
comment somewhere in here that this used to deadlock or `// verify this doesn't deadlock [anymore]` ?

Reply:
Done